training/ilab-wrapper/ilab

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
We are going to want this script installed in all bootc containers nvidia, intel and amd?

Reply:
Yes although it needs be changed to be more of a template to support that